pages/linux/qsub.md

```diff
@@ -0,0 +1,19 @@
+# qsub
+
+> Submits a script to the queue management system TORQUE on High Performance Computing clusters.
+
+- Submit a script with default settings (depends on TORQUE settings):
+
+`qsub script.sh`
+
+- Submit a script with a specified wallclock runtime of 1h 2m 3s:
+
+`qsub -l walltime=01:02:03 script.sh`
+
+- Submit a script that is executed on 2 nodes using 16 cores per node:
+
+`qsub -l nodes=2:ppn=16 script.sh`
+
+- Submit a script with a specified wallclock runtime of 1h 2m 3s to a specific queue. Note that different queues can have different maximum and minimum runtime limits:
```

Comment:
No need to add the `walltime` option here. We can just do with adding `-q` and saying `Submit a script to a given queue.`